opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/ba/FamilyMemberTestBa.java

```diff
@@ -0,0 +1,146 @@
+package org.smartregister.chw.activity.ba;
+
+import androidx.test.rule.ActivityTestRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.smartregister.chw.R;
+import org.smartregister.chw.activity.LoginActivity;
+import org.smartregister.chw.activity.utils.Constants;
+import org.smartregister.chw.activity.utils.Utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withHint;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withSubstring;
+
+public class FamilyMemberTestBa {
+
+    @Rule
+    public ActivityTestRule<LoginActivity> mActivityTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    private Utils utils = new Utils();
+
+    public void setUp() throws InterruptedException{
+        utils.logIn(Constants.BoreshaAfyaConfigUtils.ba_username, Constants.BoreshaAfyaConfigUtils.ba_password);
+        Thread.sleep(5000);
+    }
+
+    @Test
+    public void removefamilyMemberSuccessfully() throws InterruptedException{
+        onView(withSubstring(Constants.BoreshaAfyaConfigUtils.familyBa))
+                .perform(click());
+        Thread.sleep(500);
+        utils.openFamilyMenu();
+        Thread.sleep(500);
+        onView(withSubstring("Remove existing family member"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(withSubstring("Merab Emerald Nandi, 26"))
+                .perform(click());
+        onView(withSubstring("Reason for removal *"))
+                .perform(click());
+        onView(withSubstring("Moved away"))
+                .perform(click());
+        onView(withHint("Date moved away *"))
+                .perform(click());
+        onView(withSubstring("done"))
+                .perform(click());
+        onView(withId(R.id.action_save))
+                .check(matches(isDisplayed()));//perform(click());
+    }
+
+    @Test
+    public void removefamilyMemberWithoutReason() throws InterruptedException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)